app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  validates :name, presence: true,
+    uniqueness: true,
+    format: { with: /\A[a-z][a-z\s]+[a-z]\z/,
+    message: "only allows underscore letters and whitespace" }
```

Comment:
[a-z] means that there will be characters from range a-z without capital letters. Otherwise it would have to be [a-zA-Z]. rubular.com was used for creating this combination.